### PR TITLE
Switch to meta.schema from meta.topic in precache

### PR DESCRIPTION
**Pull request by halfak**
https://phabricator.wikimedia.org/T228689

### CODE REVIEW

File: tests/wsgi/tests/test_util.py [hunk 1]
```diff
@@ -72,7 +71,10 @@ def test_build_precache_map():
 def test_event():
     event = {
         "database": "enwiki",
-        "meta": {"topic": "mediawiki.revision-create"},
+        "meta": {
+            "topic": "eqiad.mediawiki.revision-create",
+            "schema": "mediawiki.revision-create"
```
Comment (ottomata) [inline tests/wsgi/tests/test_util.py]: The schema field is now at `$schema` top level.  The event will have these like:
```
$schema: /mediawiki/revision/create/1.0.0
meta: {
  stream: mediawiki.revision-create
  ...
}

File: tests/wsgi/tests/test_util.py [hunk 2]
```diff
@@ -72,7 +71,10 @@ def test_build_precache_map():
 def test_event():
     event = {
         "database": "enwiki",
-        "meta": {"topic": "mediawiki.revision-create"},
+        "meta": {
+            "topic": "eqiad.mediawiki.revision-create",
```
Comment (Pchelolo) [inline tests/wsgi/tests/test_util.py]: There will be no 'topic' in the event at all, you can delete this from the test.

File: ores/wsgi/util.py [hunk 3]
```diff
@@ -182,8 +182,7 @@ def build_event_set(event):
     uses internally.
     """
     event_set = set()
-    if re.match(r"([^\.]+.)?mediawiki\.revision-create$",
-                event['meta']['topic']):
+    if event['meta']['schema'] == "mediawiki.revision-create":
```
Comment (Pchelolo) [inline ores/wsgi/util.py]: I'm wondering if that check is needed at all. Changeprop is configured to only update ORES on `revision-create`, so nothing else could be sent here.

Comment (halfak) [reply ores/wsgi/util.py]: Eventually, we'll want to be able to precache based on other types of events.  So I'd like to keep it here.  Either way, it's good practice to check our assumption explicitly.

**Comment by Pchelolo** [comment]
LGTM

**Pull request merged**